.github/workflows/check-and-test.yaml

```diff
@@ -83,7 +83,7 @@ jobs:
         uses: DeterminateSystems/magic-nix-cache-action@test-fail-mode
```

Comment:
I think the action PR has merged, so we can stop using the `test-fail-mode` branch, right?

Reply:
Good call. Fixed.